src/hxd/fmt/hmd/Library.hx

```diff
@@ -378,6 +378,42 @@ class Library {
 	}
 	#end
 
+	// Get an unsorted list of all points in the mesh's rest position
+	// This is mainly used for collision
+	public function getStaticPoints(): Array<h3d.col.Point> {
+		var points = [];
+		var vec = new h3d.Vector();
+		for( m in header.models ) {
+			if (m.geometry < 0) continue;
+			var trans = new h3d.Matrix();
```

Comment:
I was about to say, `h3d.Matrix.I()` creates a new identity matrix, nice and easy.

Sadly they don't bother inlining it, but that can be fixed separately.

Reply:
Potato, potato.

It doesn't work as well when written.